multi-educator/src/Dscp/Educator/Web/Educator/Types.hs

```diff
@@ -0,0 +1,37 @@
+{-# LANGUAGE StrictData #-}
+
+-- | Types specific to educator API.
+
+module Dscp.Educator.Web.Educator.Types
```

Comment:
Also this, you don't need those re-exporting proxy modules, just import actual modules from `Dscp.Educator`